recipes/dominodatalab/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "dominodatalab" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dominodatalab-{{ version }}.tar.gz
+  sha256: 078e6ef8270a1e999973723287d40ea810f132dd94d8a80862f4a1ca199e529b
+
+build:
+  skip: true  # [py<37]
```

This should be a noarch recipe.
```suggestion
  noarch: python
```